app/controllers/books_controller.rb

```diff
@@ -4,4 +4,10 @@ def index
       books: Book.all
     )
   end
+
+  def show
+    @book = Book.includes(:author)
+      .joins(:author)
+      .find(params[:id])
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.